license-maven-plugin-git/src/test/java/com/mycila/maven/plugin/license/git/GitLookupTest.java

```diff
@@ -159,7 +161,7 @@ public void reuseProvider() throws GitAPIException, IOException {
   @Test
   public void timezone() throws GitAPIException, IOException {
     try {
-      new GitLookup(gitRepoRoot.toFile(), DateSource.AUTHOR, TimeZone.getTimeZone("GMT"), 10);
+      new GitLookup(gitRepoRoot.toFile(), DateSource.AUTHOR, TimeZone.getTimeZone("GMT"), 10, emptySet());
```

Comment:
Unnecessary (as are the other changes in this file) if you use an overloaded constructor.